src/main/java/org/fenixedu/learning/domain/executionCourse/ExecutionCourseSite.java

```diff
@@ -56,13 +58,16 @@
     public ExecutionCourseSite(ExecutionCourse executionCourse) {
         checkNotNull(executionCourse);
         setExecutionCourse(executionCourse);
-        setPublished(true);
+
         setFolder(folderForPath(PortalConfiguration.getInstance().getMenu(), "courses"));
         setSlug(on("-").join(getExecutionCourse().getSigla(), getExecutionCourse().getExternalId()));
-        setCanAdminGroup(NobodyGroup.get());
-        setCanPostGroup(NobodyGroup.get());
-        setBennu(Bennu.getInstance());
 
+        setCreationDate(new DateTime());
```

Comment:
Creation date is already set at the Site constructor. Calling the super() method should set that for you, so there is no need to set it here again.